pypdf/_page.py

```diff
@@ -440,12 +440,17 @@ def _get_ids_image(
         else:
             call_stack.append(_i)
         if self.inline_images_keys is None:
-            nb_inlines = len(
-                re.findall(
-                    WHITESPACES_AS_REGEXP + b"BI" + WHITESPACES_AS_REGEXP,
-                    self._get_contents_as_bytes() or b"",
-                )
-            )
+            st = self._get_contents_as_bytes() or b""
```

Comment:
Could we please have some more verbose names for `st`, `ma` and `st1` here to make the code self-documenting?